cns/Dockerfile

```diff
@@ -1,8 +1,10 @@
+# to update tag SHAs, get the latest from `skopeo inspect docker://mcr.microsoft.com/powershell:lts-nanoserver-1809 | jq -r .Digest`
```

Comment:
I had to add `--override-os windows` for this to allow me inspect the nanoserver manifest.

Also, maybe we can update this comment to point to a relevant image in the dockerfile?
```suggestion
# to update tag SHAs, get the latest from `skopeo inspect docker://mcr.microsoft.com/windows/nanoserver:ltsc2019 | jq -r .Digest`
```

Reply:
Also, this would be useful if this comment was in all 3 dockerfiles 😄

Reply:
moving to the FROM lines in a more helpful format